tests/Unit/UserTest.php

```diff
@@ -87,8 +94,70 @@ public function testLastLogin()
 
         $last = $this->user->lastLogin();
 
-        $this->assertInstanceOf(Login::class, $last); // @phpstan-ignore-line
+        $this->assertInstanceOf(Login::class, $last);
```

Comment:
Revert, please.

```
GitHub Actions / PHP 7.4 Static Analysis

tests/Unit/UserTest.php#L97

Call to method PHPUnit\Framework\Assert::assertInstanceOf() with 'CodeIgniter\\Shield\\Entities\\Login' and null will always evaluate to false.
```

Reply:
Same as the other PR 😂 I think Lonnie has an outdated local installation of PHPStan

Reply:
Damn you Stan. :)